### PR TITLE
DBZ-3038 Capture additional LogMiner JMX metrics

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3038

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java [hunk 1]
```diff
@@ -290,20 +315,44 @@ public void addCurrentParseTime(Duration currentParseTime) {
...
     @Override
     public long getTotalProcessingTimeInMilliseconds() {
         return totalProcessingTime.get().toMillis();
     }
 
-    public void setTotalProcessingTime(Duration processingTime) {
-        totalProcessingTime.set(processingTime);
+    @Override
+    public long getMinBatchProcessingTimeInMilliseconds() {
+        return 0;
```
Comment (gunnarmorling) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java:346]: Is 0 expected here?

**Review by gunnarmorling — COMMENTED** [review]
One question inline; and does it need a docs update?

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java (hunk 1 above)
Comment (Naros) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java]: Ouch, my mistake.  I fixed & repushed.

**Comment by Naros** [comment]
> docs update?

Yes indeed, that's my next step with this to make sure the docs are synced with all our exposed metrics.

**Comment by Naros** [comment]
Metrics documentation synchronized in https://github.com/debezium/debezium/pull/2161

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Nice!

**Pull request merged**